l1-contracts/contracts/state-transition/chain-deps/facets/Admin.sol

```diff
@@ -398,7 +395,7 @@ contract AdminFacet is ZKChainBase, IAdmin {
         // Set the settlement to 0 - as this is the current settlement chain.
         s.settlementLayer = address(0);
 
-        _setDAValidatorPair(address(0), address(0));
+        _setDAValidatorPair(address(0), L2DACommitmentScheme.EMPTY_NO_DA);
```

Comment:
Should it not be None? Empty_no_da is for validiums, that different than address(0)

Reply:
Agree, it should be None